source/authors/index.html.slim

```diff
@@ -0,0 +1,19 @@
+---
+layout: landing
+---
+
+section
+  .flex-posts-list
+    - authors.each do |author|
```

Comment:
Looks fine, but personally I prefer extracting the template code for a single item into a partial, but that's a personal preference.

Reply:
added
```
= partial 'partials/author-teaser', locals: { author: author, post_count:  post_count }
```